### DISCUSSION TITLE
Getting the status of a badge from a private repository

### DISCUSSION THREAD

**Discussion by ccamacho** [community/community#26567] [Actions]
<p>Hi,</p>
<p>I’m trying to get the status of a badge in a private repository, I’m doing this from an unauthenticated location and I’m receiving the expected 404 error message.</p>
<p>Can you tell me how should I add the access_token information to the GET method to actually be able to get its status?</p>
<p>Something like: <a href="https://github.com/xx/yy/workflows/zz/badge.svg?event=push&amp;access_token=hhhh" rel="nofollow noopener">https://github.com/xx/yy/workflows/zz/badge.svg?event=push&amp;access_token=hhhh</a><br>
Still returns 404.</p>
<p>Also setting the token in the header in the request:</p>
<p>headers: {‘Authorization’: ‘token hhhh’,<br>
Still returns 404.</p>
<p>Thanks!</p>

**Comment by ghost** [comment, marked as answer]
<p>Unfortunately private badge statuses are not exposed with our current URL structure. Sorry about that!</p>
<p>We are working to update the documentation to reflect the limitations of accessing a private badge status from outside the repository:</p>
<p><a href="https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository" class="onebox" target="_blank">https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository</a></p>
<p>We really appreciate feedback on how we can make GitHub even better and the best way to report any feature requests is directly to us through our feedback form: <a href="https://support.github.com/contact/feedback">https://support.github.com/contact/feedback</a></p>
<p>Our roadmap is not publicly visible, so we recommend that you keep an eye on the <a href="https://github.com/blog">GitHub Blog</a> for the latest announcements about new features.</p>

**Comment by KevinGDialpad** [comment]
<p>I was trying to do the same thing, and it took me some time to realize that it isn’t possible.<br>
The docs have moved to <a href="https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge" class="inline-onebox" rel="noopener nofollow ugc">Adding a workflow status badge - GitHub Docs</a>, but they haven’t been updated. It would be great if they mentioned that limitation.</p>
<p>I posted a feature request as well.</p>
